build/steps-codecheck.yml

```diff
@@ -0,0 +1,46 @@
+##
+# Run all build steps.
+##
+steps:
+- template: steps-init-codecheck.yml    # Win: Install choco
+ 
+  # QIR Runtime:
+- pwsh: src/Qir/Runtime/prerequisites.ps1
+  displayName: "Install LLVM, ninja cmake"
```

Comment:
This may not remain accurate if we update prerequisites.ps1.

Reply:
Changed to `displayName: "Install QIR Runtime Prerequisistes"`